### PR TITLE
Implement persistent speed setting.

### PR DESCRIPTION
**Pull request by sjonany**

### CODE REVIEW

**Comment by sjonany** [comment]
Tested that speed is persisted across tabs and browser restart.
Minor tweak in the reading display so the speed reflects autoscroll speed.

File: extension/lib/display.js [hunk 1]
```diff
@@ -28,11 +28,11 @@ class Display {
         this.readableDomIds = readableDomIds; // Used to calc initial reading time
         this.html = null;
         this.time_remaining = this.initTimer(total_words);
-        this.reading_speed = this.initSpeed(speed);
         this.end = null; 
 
         this.defineHtml();
         this.createDisplay(readableDomIds);
+        this.updateSpeed(speed);
```
Comment (sjonany) [inline extension/lib/display.js:35]: So there is a behavior change I introduced here.
Now I make sure that readingSpeed = a function of (content_script.js speed) at all time.

I can demo to show how this feels / looks like

Comment (benfrancis314) [reply extension/lib/display.js:35]: Great! When I finish similar reading time stuff we can dual-demo

**Review by benfrancis314 — APPROVED** [review]
LGTM

**Pull request merged**